package.json

```diff
@@ -80,8 +81,9 @@
     "slugify": "^1.6.5",
     "swr": "^2.2.4",
     "uuidv7": "^0.6.3",
-    "viem": "^2.9.6",
-    "wagmi": "^2.5.17",
+    "viem": "2.9.6",
```

Comment:
encountered a type error in newer versions, so I fixed the versions for these packages